### PR TITLE
start fixing up windows hints

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

File: chia/server/chia_policy.py [hunk 1]
```diff
@@ -275,7 +275,7 @@ async def _chia_accept_loop(self, listener: socket.socket) -> Tuple[socket.socke
 
         def _chia_accept(self, listener: socket.socket) -> asyncio.Future[Tuple[socket.socket, Tuple[object, ...]]]:
             self._register_with_iocp(listener)
-            conn = self._get_accept_socket(listener.family)
+            conn = self._get_accept_socket(listener.family)  # pylint: disable=assignment-from-no-return
```
Comment (arvidn) [inline chia/server/chia_policy.py:277]: is mypy mistaken in thinking that `_get_accept_socket()` doesn't return anything?

**Comment by arvidn** [comment]
looks reasonable so far. there seem to be some legitimate failures in CI: https://github.com/Chia-Network/chia-blockchain/actions/runs/4169394633/jobs/7218892064#step:16:239

File: chia/server/chia_policy.py (hunk 1 above)
Comment (altendky) [reply chia/server/chia_policy.py:277]: pylint is the confused one.  It doesn't special case the `typing.TYPE_CHECKING` afaik so would treat it as the `False` it is coded to for runtime.  This means it misses all of our hinting shims above.  There may be more to look at here.

**Comment by altendky** [comment]
otherwise made their way into https://github.com/Chia-Network/chia-blockchain/pull/14575

**Pull request closed without merging**